zstacklib/zstacklib/utils/file_metadata_handler.py

```
@@ -0,0 +1,244 @@
+        logger.debug("scan_vm_metadata on %s: found %d entries",
+                     metadata_dir, len(entries))
+        return {'metadataEntries': entries}
+
+    def _do_cleanup(self, cmd):
+        metadata_path = cmd.metadataPath
+
+        # delete .json
+        try:
+            if os.path.exists(metadata_path):
+                os.remove(metadata_path)
+        except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)